packages/twenty-front/src/modules/workspace/components/WorkspaceMemberCard.tsx

```diff
@@ -28,30 +31,50 @@ const StyledEmailText = styled.span`
 `;
 
 type WorkspaceMemberCardProps = {
-  workspaceMember: WorkspaceMember;
+  workspaceMember: WorkspaceMember | WorkspaceInvitation;
   accessory?: React.ReactNode;
 };
 
+const isWorkspaceInvitationGuard = (
```

Comment:
I don't get this ; I don't understand how the expiresAt existence would tell it's a workspaceInvitation

Reply:
maybe it's just a naming issue: isWorkspaceMemberInvited?

Reply:
but still checking for expiresAt since flaky

Reply:
This component doesn't exist anymore